File: Client/Classes/PluginHost.vb
Public Class PluginHost

    Implements Versidyne.Plugins.Host

    Public Sub ShowFeedback(ByVal strFeedback As String) Implements Versidyne.Plugins.Host.ShowFeedback

        MessageBox.Show(strFeedback, Application.ProductName)

    End Sub

End Class

File: Client/Classes/PluginHost.cs
// VBConversions Note: VB project level imports
using System.Collections.Generic;
using System;
using System.Linq;
using System.Drawing;
using System.Diagnostics;
using Microsoft.VisualBasic;
using System.Xml.Linq;
using System.Collections;
using System.Windows.Forms;
// End of VB project level imports


namespace Vexis
{
	public class PluginHost : Versidyne.Plugins.Host
	{
		
		
		public void ShowFeedback(string strFeedback)
		{
			
			MessageBox.Show(strFeedback, Application.ProductName);
			
		}
		
	}
	
}
